onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -171,6 +171,18 @@ def _amax_amin_input_wrangler(
     return args, kwargs
 
 
+def _arange_input_wrangler(
```

Comment:
Why move from previous fix to adding 'wrangler', given this is a general issue not specific to arange?
Are you planning a more fundamental fix in onnx-script?